components/gradle.properties

```diff
@@ -9,7 +9,7 @@ android.useAndroidX=true
 #Versions
 kotlin.version=2.1.0
 agp.version=8.2.2
-compose.version=1.8.0-alpha01
+compose.version=1.8.0-alpha03
```

Comment:
from https://github.com/JetBrains/compose-multiplatform/pull/5219#issuecomment-2677321021

> Now there is only failed test on GitHub CI due to MissingResourceException, which is because CI does not use latest version of resources component library, it has changes which are required for the test to succeed.

If you need a new functionality from Compose, please update the version in `gradle.properties` to the latest dev, it is fine to use it (`1.8.0+dev2134` at the moment)